python/src/main/python/yaml-template/requirements.txt

```diff
@@ -1006,13 +885,13 @@ jsonpickle==3.4.2 \
     --hash=sha256:2efa2778859b6397d5804b0a98d52cd2a7d9a70fcb873bc5a3ca5acca8f499ba \
     --hash=sha256:fd6c273278a02b3b66e3405db3dd2f4dbc8f4a4a3123bfcab3045177c6feb9c3
     # via apache-beam
-jsonschema==4.25.1 \
-    --hash=sha256:3fba0169e345c7175110351d456342c364814cfcf3b964ba4587f22915230a63 \
-    --hash=sha256:e4a9655ce0da0c0b67a085847e00a3a51449e1157f4f75e9fb5aa545e122eb85
+jsonschema==4.23.0 \
```

Comment:
Beam 2.68 already uses:

https://github.com/apache/beam/blob/release-2.68/sdks/python/container/py312/base_image_requirements.txt#L104

Please check why it pins down the old packages here. 

cc @jrmccluskey  @Amar3tto